operators/cloud-native-postgresql/1.22.3/manifests/cloud-native-postgresql.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1283 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "postgresql.k8s.enterprisedb.io/v1",
+          "kind": "Backup",
+          "metadata": {
+            "name": "backup-sample"
+          },
+          "spec": {
+            "cluster": {
+              "name": "cluster-sample"
+            }
+          }
+        },
+        {
+          "apiVersion": "postgresql.k8s.enterprisedb.io/v1",
+          "kind": "Cluster",
+          "metadata": {
+            "name": "cluster-sample"
+          },
+          "spec": {
+            "instances": 3,
+            "logLevel": "info",
+            "primaryUpdateStrategy": "unsupervised",
+            "storage": {
+              "size": "1Gi"
+            },
+            "walStorage": {
+              "size": "1Gi"
+            }
+          }
+        },
+        {
+          "apiVersion": "postgresql.k8s.enterprisedb.io/v1",
+          "kind": "Pooler",
+          "metadata": {
+            "name": "pooler-sample-rw"
+          },
+          "spec": {
+            "cluster": {
+              "name": "cluster-sample"
+            },
+            "instances": 1,
+            "pgbouncer": {
+              "poolMode": "session"
+            },
+            "type": "rw"
+          }
+        },
+        {
+          "apiVersion": "postgresql.k8s.enterprisedb.io/v1",
+          "kind": "ScheduledBackup",
+          "metadata": {
+            "name": "scheduledbackup-sample"
+          },
+          "spec": {
+            "cluster": {
+              "name": "cluster-sample"
+            },
+            "schedule": "0 0 0 * * *"
+          }
+        }
+      ]
+    capabilities: Auto Pilot
+    categories: Database
+    certified: "true"
+    containerImage: quay.io/enterprisedb/cloud-native-postgresql@sha256:cf211142acd551d8a925914efa7653a5781fc8950c5a093caab914b833472914
+    createdAt: "2024-04-25T10:22:39Z"
+    description: EDB Postgres for Kubernetes is an open source operator designed to manage highly available PostgreSQL databases with a primary/standby architecture on any supported Kubernetes cluster.
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "true"
+    features.operators.openshift.io/proxy-aware: "false"
+    features.operators.openshift.io/tls-profiles: "false"
+    features.operators.openshift.io/token-auth-aws: "false"
+    features.operators.openshift.io/token-auth-azure: "false"
+    features.operators.openshift.io/token-auth-gcp: "false"
+    olm.skipRange: '>= 1.18.0 < 1.22.3'
```

Comment:
use skips in place of olm.skipRange, as skip is an array and must be formatted properly

skips:
   - cloud-native-postgresql.v1.22.2

Reply:
@itsmitul9 Thanks for the comments. 
Can I assume once skips is used in 1.22.3 operator, the olm upgrade could cover all previous version in semer range '>= 1.18.0 < 1.22.3'. 

If yes, is the upgrade sequence is

As in head of channel we have skips and replaces defined. 

skips: 
- cloud-native-postgresql.v1.22.2

and 
replaces:  cloud-native-postgresql.v1.22.2

then in v.1.22.2, we have skipsRange defined.
skipsRange:  '>= 1.18.0 < 1.22.2'

So , if the current install operator falls in  '>= 1.18.0 < 1.22.2',  OLM will update it to latest in the channel. 

if the current install operator is 1.22.2, meet the skip in 1.22.3, OLM will update it to the latest in the channel. 

Please correct me if am wrong.

Reply:
Hi @itsmitul9 

We haven't told like that by @jfrancin in a ticket we opened with the Partner Support, we actually want to keep all the versions in the catalog and not just a few, ones, can you elaborate on how this will work and behave in the future? 

Thank you

Reply:
The dev team has been adding lots of static checks to the pipeline of late.

One of the changes - which I've reported as a bug - is to flag 'replaces' clauses in _previous versions of an operator_ - which cannot be fixed - as being erroneous. 

I will add the appropriate flag to skip that test and requeue the bundle.